README.md

```diff
@@ -7,10 +7,30 @@ The best way to use it is via [vaadin.com/start](https://vaadin.com/start) - you
 
 To access it directly from github, clone the repository and import the project to the IDE of your choice as a Maven project. You need to have Java 8 or 11 installed.
 
-Run using `mvn -Pprepare-osgi-container -Prun-osgi-container -Pproduction verify` and open [http://localhost:8080](http://localhost:8080) in browser.
-This command will run profiles which downloads for you Felix OSGi container and installs `felix-jetty` 
-as OSGi web container. It deploys all necessary bundles to Felix and start it synchronously.
-The container will be stopped  when you stop your maven build (e.g. by killing it via `Ctrl+C`).
+Main Command: `mvn clean install`
+- build all bundles
+- run tests inside the osgi framework when `tests.bndrun` file exists in project.
+- generates the Application if `application.bndrun` file exists in project.
+
+Run the Application using `java -jar application/application.jar`
+
+
+Special Commands:
+- Resolve the Framework using `mvn bnd-resolver:resolve application/pom.xml`
```

Comment:
This command simply fails just because 

```
No plugin found for prefix 'bnd-resolver' in the current project and in the plugin groups
```

Reply:
mvn bnd-resolver:resolve -f application/pom.xml

Reply:
Unfortunately this goal is quite cryptic: the sentence "it resolves" doesn't say anything to the end user.

It resolves: yes. But for me it meant that the bundles are printed in the console and that's it.
Apparently :
- if I have `-runbundles:` in the bndrun file then the resolution happens temporary only: it prints the resolved bundles in the log and that's it.
- I had to remove `-runbundles:` from bndrun file completely and only after that I see that the goal has written the resolved bundles into the bndrun.

This is extremely confusing and undocumented behavior :(